index.html

```diff
@@ -3,20 +3,54 @@
   <head>
     <meta charset="utf-8">
     <title>ACLU Cash Bail</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/components/transition.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/components/dropdown.min.css">
     <link rel="stylesheet" href="src/css/style.css">
     <link rel="stylesheet" href="src/css/table.css">
+    <link rel="stylesheet" href="src/css/search.css">
   </head>
   <body>
     <div class="container">
-      <table id="bail-rate-table">
-        <thead></thead>
-        <tbody></tbody>
-      </table>
-      <table id="bail-race-table">
-        <thead></thead>
-        <tbody></tbody>
-      </table>
-    </div>
-    <script src="src/js/index.js" charset="utf-8"></script>
+      <div id="bail-rate-container">
+        <div class="ui fluid multiple search selection dropdown">
+          <input type="hidden" name="county">
+          <i class="dropdown icon"></i>
+          <div class="default text">Select County</div>
+          <div class="menu"></div>
+        </div>
+        <table id="bail-rate-table">
+          <thead></thead>
+          <tbody></tbody>
+        </table>
+      </div>
+      <div id="bail-race-container">
+        <div class="ui fluid multiple search selection dropdown">
+          <input type="hidden" name="county">
+          <i class="dropdown icon"></i>
+          <div class="default text">Select County</div>
+          <div class="menu"></div>
+        </div>
+        <table id="bail-race-table">
+          <thead></thead>
+          <tbody></tbody>
+        </table>
+      </div>
   </body>
+    <script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.slim.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/components/transition.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/components/dropdown.min.js"></script>
+    <script src="src/js/index.js" charset="utf-8"></script>
+    <script>
+      $(document).ready(() => {
```

Comment:
Based on this it doesn't look like jQuery's needed. We should be able to just put this at the end of the table's render function, no?

Reply:
If I understand the code right, it seems like the dropdown() function requires jQuery (I could be wrong). Like, document.getElementsByClassName("dropdown")[0].dropdown() doesn't work because it seems like the dropdown() function is registed to the jQuery node.

If this needs jQuery, I would rather we keep all of our render functions as pure jS and isolate the jQuery.